components/FeaturedCarousel.js

```diff
@@ -0,0 +1,54 @@
+import { useContext } from "react";
+import IssuesContext from "../contexts/IssuesContext";
+import IndexIsssueListItem from "../components/shared/IndexIssueListItem";
+import { Row, Col, Carousel } from "react-bootstrap";
```

Comment:
i think nextjs does a good job of optimizing this for you, but you should import each component using its default export. There's a lil blurb about it in the docs, [here](https://react-bootstrap.github.io/getting-started/introduction/#importing-components).

```javascript
import Row from "react-bootstrap/Row";
import Col from "react-bootstrap/Carousel";
import Carousel from "react-bootstrap/Carousel";
```